tests/test_rest.py

```diff
@@ -541,6 +541,78 @@ def test_data(reqmock):
     assert type(quote) == tradeapi.entity.Quote
     assert type(api_raw.get_last_quote('AAPL')) == dict
 
+    # Latest trade
+    reqmock.get(
+        'https://data.alpaca.markets/v2/stocks/AAPL/trades/latest',
+        text='''
+        {
+			"symbol": "AAPL",
+			"trade": {
+				"t": "2021-04-20T12:40:34.123456789Z",
+				"x": "J",
+				"p": 134.7,
+				"s": 20,
+				"c": [
+					"@",
+					"T",
+					"I"
+				],
+				"i": 32,
+				"z": "C"
+			}
+		}
+        '''
+    )
+    latest_trade = api.get_latest_trade('AAPL')
+    assert latest_trade.exchange == "J"
+    assert latest_trade.price == 134.7    
+    assert latest_trade.size == 20
+    assert latest_trade.conditions == ["@","T","I"]
```

Comment:
[minor] missing spaces 
```suggestion
    assert latest_trade.conditions == ["@", "T", "I"]
```